sdcm/utils/url_utils.py

```diff
@@ -0,0 +1,89 @@
+import re
+from collections import namedtuple
+from urllib.request import urlopen
+
+from sdcm.utils.threads_utils import ThreadWithReturnValue
+
+SCYLLA_URL_REGEX = re.compile('.*(?P<url>http.*scylladb.com.*).*')
```

Comment:
again, why are we limiting just to scylladb.com ?

this should work for any reachable urls....
event if I host my own repo on my laptop, and give the public address of it.

Reply:
but you are always validating the URL, and we won't be able to run such repos, pointing to something which is not scylladb.com

Reply:
I still don't understand why do we need to distinguish between Scylla and non-Scylla URLs